### ISSUE TITLE
Bug: Safari keyboard navigation does not work

### ISSUE THREAD

**Issue by sk-roofr** [gpbl/react-day-picker#1752]
### Bug description

When I open the demo page in Safari, the keyboard navigation does not work.

MacOs: Ventura 

### To reproduce

Fork this CodeSandbox: https://codesandbox.io/s/react-day-picker-8-vl4ry with the code to reproduce the issue.

It does not work with any keyboard navigation.

### Expected behavior
Arrow and date get highlighted when the user presses the tab.
